src/components/app/App.jsx

```diff
@@ -6,7 +6,6 @@ import moment from 'moment'
 
 import BoardsContainer from '../../containers/boards/BoardsContainer'
 import LoginContainer from '../../containers/login/LoginContainer'
-import LogoutContainer from '../../containers/logout/LogoutContainer'
```

Comment:
Why deleting Logout ?

Reply:
It's now handle inside the User container